docs/_docs/plugins.md

```diff
@@ -59,13 +64,14 @@ values of the gem names of the plugins you'd like to use. An example:
   </p>
 </div>
 
-In general, plugins you make will fall into one of five categories:
+In general, plugins you make will fall broadly into one of five categories:
 
 1. [Generators](#generators)
 2. [Converters](#converters)
 3. [Commands](#commands)
 4. [Tags](#tags)
 5. [Hooks](#hooks)
+6. [List of available plugins](#available-plugins)
```

Comment:
This isn't a category, so this link should go elsewhere. Additionally, we should really move this list out of this page – it's clutter! /cc #5771